lib/simple-dom/html-serializer.js

```diff
@@ -23,14 +38,11 @@ HTMLSerializer.prototype.attributes = function(namedNodeMap) {
 };
 
 HTMLSerializer.prototype.escapeAttrValue = function(attrValue) {
-  return attrValue.replace(/[&"]/g, function(match) {
```

Comment:
did you try making the regex and the replacer a constant instead of allocated every escapeAttrValue?